core/src/main/java/dk/alexandra/fresco/framework/network/KryoNetNetwork.java

```diff
@@ -108,31 +125,31 @@ public ClientConnectThread(Client client, String hostname, int port, Semaphore s
       this.semaphore = semaphore;
     }
 
+    @Override
     public void run() {
       boolean success = false;
       int maxRetries = 30;
+      int sleepTime = 200;
       int retries = 0;
       while (!success) {
         try {
           retries++;
-          client.connect(2000, hostname, port);
+          client.connect(1000, hostname, port);
```

Comment:
One second connection is very reasonable for unittests, but in real life it is very little time. Maybe defer it to a parameter?